core/src/input_method.rs

```diff
@@ -34,15 +34,20 @@ pub struct Preedit<T = String> {
     pub content: T,
     /// The selected range of the content.
     pub selection: Option<Range<usize>>,
+    /// The text size of the content.
+    pub text_size: Option<Pixels>,
 }
 
 impl<T> Preedit<T> {
     /// Creates a new empty [`Preedit`].
-    pub fn new() -> Self
+    pub fn new(text_size: Option<impl Into<Pixels>>) -> Self
     where
         T: Default,
     {
-        Self::default()
+        Self {
+            text_size: text_size.map(Into::into),
+            ..Default::default()
+        }
```

Comment:
No need. Let's not ask for any argument and just use `Default`.

Reply:
Sure. I'll make a change.

Reply:
@hecrj I fixed this at cf851e133ad6aaedaf07b58c68e7c41d41ee151a